manifests/mc2.pp

```diff
@@ -0,0 +1,51 @@
+# == Class: seed_stack::mc2
+#
+# Runs a Mission Control container in Marathon.
+#
+class seed_stack::mc2 {
+
+  unless defined(Package['curl']) {
+    package { 'curl': ensure => 'installed' }
+  }
+
+  file { '/usr/local/sbin/manage-marathon-group.sh':
```

Comment:
No real reason. I probably just saw something else that put a script in `/usr/local/sbin` and did the same.